develop/items/first-item.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Creating Your First Item
+description: Learn how to register a simple item and how to texture, model and name it.
+authors:
+  - IMB11
+---
+
+# Creating Your First Item
+
+This page will introduce you into some key concepts relating to items, and how you can register, texture, model and name them.
+
+If you aren't aware, everything in Minecraft is stored in registries, and items are no exception to that.
+
+## Preparing Your Items Class
+
+To simplify the registering of items, you can create a method that accepts an instance of an item and a string identifier.
+
+You can put this method in a class called `ModItems` (or whatever you want to name the class!).
+
+Mojang does this with their items as well! Check out the `Items` class for inspiration.
+
+@[code transcludeWith=:::1](@/reference/latest/src/main/java/com/example/docs/item/ModItems.java)
+
+## Registering an Item
+
+You can now register an item using the method now.
+
+The item constructor takes in an instance of the `Items.Settings` class as a parameter. This class allows you to:
+
+- Assign the item's `ItemGroup`.
+- Make the item edible by passing a `FoodComponent`.
+- Set the item's durability.
+- ...and other miscellaneous properties.
+
+::: tip
+If you want to change your item's stack size, you can use the `maxCount` method in the `Items.Settings`/`FabricItemSettings` class.
+
+This will not work if you've marked the item as damageable, as the stack size is always 1 for damageable items to prevent duplication exploits.
+:::
+
+@[code transcludeWith=:::2](@/reference/latest/src/main/java/com/example/docs/item/ModItems.java)
+
+However, when you go in-game, you can see that our item doesn't exist! This is because you don't statically initialize the class.
```

Comment:
As I said bellow im unsure if having it a seperate class really helps.

Reply:
Its mostly to encourage modularization, i've seen many new mod developers have 100s of items stored in their mod initializers.